src/components/Navbar.js

```diff
@@ -23,30 +22,18 @@ const Navbar = props => (
       </button>
       <div className="collapse navbar-collapse " id="navbarNav">
         <ul className="navbar-nav ml-auto">
-
-          { props.loggedIn ?
+          {
+            (props.loggedIn === true) && (
```

Comment:
if props.loggedIn is a boolean, use props.loggedIn&&(......) . instead of props.loggedIn=== true&&(....)

Reply:
Oh you are right but I did that ===true because it helped solve some error being caused by redux persist but take it us a quick hack that might be cleared after implement some other functionalities

Reply:
I have actually resolved the error thank you